src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  --placeholder-color: #3d4e61;
+}
+
+body {
+  font-weight: 300;
+  margin: 0;
+}
+
+.container {
+  padding: 20px 8px;
+}
+
+.form:first-child {
+  margin-bottom: 20px;
+}
+
+.search {
+  font-weight: 300;
+  font-family: inherit;
+  width: 100%;
+
+  display: block;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  outline: none;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search--big {
+  height: 70px;
+  padding: 20px 0;
+  text-indent: 62px;
+  font-size: 16px;
+  font-weight: 300;
```

Comment:
Why do you need to repeat font-weight in .search class and then in .search-big
Read checklist 2